### DISCUSSION TITLE
How to delete categories background.

### DISCUSSION THREAD

**Discussion by StopDragon** [CaiJimmy/hugo-theme-stack#503] [Q&A] [answered]
Hello. I'm using your theme very well.
Thank you very much.

<img width="665" alt="스크린샷 2022-02-24 22 48 49" src="https://user-images.githubusercontent.com/43020330/155536286-21ea2fef-0279-4b67-a20d-c3cb9aee4a49.png">

I want to subtract the background color of the categories here.

<img width="306" alt="스크린샷 2022-02-24 22 45 03" src="https://user-images.githubusercontent.com/43020330/155536128-f315c727-2641-4810-b528-9c1b151af2e6.png">

It seems to be this code, but it doesn't go away even if I delete it.

`/layouts/partials/article/components`

 I have followed the code below in , but it doesn't go away. please tell me how thank you.

` <a href="{{ .RelPermalink }}" {{ with .Params.style }}style="background-color: {{ .background }}; color: {{ .color }};"{{ end }}>`

**Comment by CaiJimmy** [comment, marked as answer]
The code is located in `main.ts`:

https://github.com/CaiJimmy/hugo-theme-stack/blob/0fbbc12949c8d59ea5ddcfb463d2c48140bfb490/assets/ts/main.ts#L30-L59

**Reply by StopDragon** [reply]
Oh! I got it, Thanks 👍
